core/src/main/java/com/linecorp/armeria/client/Client.java

```diff
@@ -51,16 +52,26 @@
     O execute(ClientRequestContext ctx, I req) throws Exception;
 
     /**
-     * Undecorates this {@link Client} to find the {@link Client} which is an instance of the specified
-     * {@code clientType}.
+     * Unwraps this {@link Client} into the object of the specified {@code type}.
+     * Use this method instead of an explicit downcast. For example:
+     * <pre>{@code
+     * HttpClient client = new HttpClientBuilder()
+     *     .decorator(LoggingClient.newDecorator())
+     *     .build();
+     * LoggingClient unwrapped1 = client.as(LoggingClient.class).get();
+     * LoggingClient unwrapped2 = Clients.unwrap(client, LoggingClient.class).get();
```

Comment:
I find this example a little confusing since they look the same and I don't know which one to use. Would it make sense to instead mention explicitly - `for gRPC and Thrift clients, see Clients.unwrap`?

Reply:
Good idea. Updated.